accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/authorizableutils/AuthorizableConfigBean.java

```diff
@@ -28,7 +28,6 @@
     private String[] members;
     String membersStringFromConfig;
 
-    private String description;
```

Comment:
This was not used anywhere - we should introduce a more generic approach for setting profile information (created #71)